barretenberg/cpp/src/barretenberg/stdlib/special_public_inputs/special_public_inputs.hpp

```diff
@@ -12,9 +12,6 @@
 #include "barretenberg/stdlib/primitives/public_input_component/public_input_component.hpp"
```

Comment:
Note: I could do a bit more logic sharing in this file. I'll simplify in a follow on